pgtype/interval.go

```diff
@@ -27,10 +28,11 @@ type IntervalValuer interface {
 }
 
 type Interval struct {
-	Microseconds int64
-	Days         int32
-	Months       int32
-	Valid        bool
+	Microseconds     int64
+	Days             int32
+	Months           int32
+	InfinityModifier InfinityModifier
```

Comment:
Is it worth adding a field comment here along the lines of this?

```golang
// InfinityModifier for intervals is only supported in Postgres 17 or newer
InfinityModifier InfinityModifier
```

Reply:
Wouldn't hurt.